test/rtest-gdk-events.lisp

```diff
@@ -119,7 +119,16 @@
                 (:TOUCH-UPDATE 38)
                 (:TOUCH-END 39)
                 (:TOUCH-CANCEL 40)
-                (:EVENT-LAST 41))
+                #+gdk-3-18 (:TOUCHPAD-SWIPE 41)
+                #+gdk-3-18 (:TOUCHPAD-PINCH 42)
+                #+gdk-3-22 (:PAD-BUTTON-PRESS 43)
+                #+gdk-3-22 (:PAD-BUTTON-RELEASE 44)
+                #+gdk-3-22 (:PAD-RING 45)
+                #+gdk-3-22 (:PAD-STRIP 46)
+                #+gdk-3-22 (:PAD-GROUP-MODE 47)
+                (:EVENT-LAST #-gdk-3-18 41
```

Comment:
Honestly I'd rather throw this out than update it for `:event-last` all the time.